src/transport/raw/TCP.cpp

```diff
@@ -39,11 +39,11 @@ namespace {
 
 using namespace chip::Encoding;
 
-// Packets start with a 16-bit size
-constexpr size_t kPacketSizeBytes = 2;
+// Packets start with a 32-bit size field.
+constexpr size_t kPacketSizeBytes = sizeof(uint32_t);
```

Comment:
Please leave this as "4".  Nothing in theory guarantees that sizeof() is measured in units of 8 bits in C/C++ (though of course a lot of our code would break if it wasn't).